### PR TITLE
Loading spinners

### PR DESCRIPTION
**Pull request by mattheu**
image

### CODE REVIEW

**Comment by mattheu** [comment]
How do these look? Is there a neater way to do them?

File: src/styles/components/_Loading.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	background: url( "#{ $hm-images }/icons/icon-spinner-black.svg" ) no-repeat center center;
+	background-size: 100% 100%;
+	animation-name: spin;
+	animation-duration: 1500ms;
+	animation-iteration-count: infinite;
+	animation-timing-function: linear;
+	display: inline-block;
+	margin-right: $gutter-width / 2;
+	position: relative;
+	top: 4px;
```
Comment (sambulance) [inline src/styles/components/_Loading.scss]: This looks quite specific - is this more related to the styleguide?

**Comment by sambulance** [comment]
I think the way you've done this is fine, the only other way would be an animated gif instead of CSS rotation. Perhaps we can add a class to center and element using `transform()`? (I think that's what the `top` was for?)

File: src/styles/components/_Loading.scss (hunk 1 above)
Comment (mattheu) [inline src/styles/components/_Loading.scss]: Yeah I think you're right. And I like the idea of a centered class.

**Comment by mattheu** [comment]
I played with the code a bit for this.  I think it would be most useful if this was the size of the base line height, and that the spinner was centered within this. It does still need a little offset though.

**Comment by mattheu** [comment]
I'd rather avoid gifs - just because I don't want to faff about with different size images for different screen resolutions and stuff like that :)

**Comment by sambulance** [comment]
Oh definitely agree to avoid gifs :) Base line height is a good reference point. Looks good :)
